### PR TITLE
remove CALLC and DEPX from VM

### PR DESCRIPTION
**Pull request by zora-wang**
@junmendoza . 
CALLC and DEPX are removed

relevant task is here : http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7074

### CODE REVIEW

**Comment by junmendoza** [comment]
Hi @zora-wang there are also CALLC and DEPX code in Executive.cs.
They also need to be removed :)

**Comment by zora-wang** [comment]
Hi, @junmendoza . I have updated files.please review it again~~

**Comment by junmendoza** [comment]
LGTM

File: src/Engine/ProtoCore/DSASM/Executive.cs [hunk 1]
```diff
@@ -7478,11 +7466,7 @@ private void Exec(Instruction instruction)
                         return;
                     }
 
-                case OpCode.DEPX:
```
Comment (junmendoza) [inline src/Engine/ProtoCore/DSASM/Executive.cs]: @zora-wang Theres one more, pls remove this: 
private void DEPX_Handler()

Comment (zora-wang) [inline src/Engine/ProtoCore/DSASM/Executive.cs]: Hey @junmendoza . DEPX_Handler() has been removed

Comment (junmendoza) [inline src/Engine/ProtoCore/DSASM/Executive.cs]: thanks

**Pull request merged**